README.md

```diff
@@ -17,21 +17,20 @@ The project will contain:
 ## Requirements ##
 
 * Python 3.6+
-* pip/pipenv (used to get the rest of the requirements)
+* pip/poetry (used to get the rest of the requirements)
 
 ## Install ##
 
 ```bash
-cd kiso-testing
-pip install .
+pip install pykiso
 ```
 
-[Pipenv](https://github.com/pypa/pipenv) is more appropriate for developers as it automatically creates virtual environments.
+[Poetry](https://python-poetry.org/) is more appropriate for developers as it automatically creates virtual environments.
 
 ```bash
 cd kiso-testing
-pipenv install --dev
-pipenv shell
+poetry install
```

Comment:
we are in thee user installation s maybe the dev dependencies are not needed and user has to write :
poetry install --no-dev

Reply:
I agree for the getting_started, but here we state "is more appropriate for developers", so from what I understan this part would be for developers/contributors

Additionnally, we previously had pipenv install --dev here